guides/common/modules/proc_registering-hosts-for-conversion.adoc

```diff
@@ -0,0 +1,23 @@
+[id="registering-hosts-for-conversion_{context}"]
+
+= Registering Hosts for Conversion
+
+Use this procedure to register the hosts to {Project}.
```

Comment:
This would IMHO make it clearer:

Use this procedure to register CentOS hosts to {Project}.

Reply:
Fixed